lightning/src/ln/channelmanager.rs

```diff
@@ -333,29 +337,173 @@ mod inbound_payment {
 		}
 		return Ok(PaymentPreimage(decoded_payment_preimage))
 	}
+}
+
+/// LDK has multiple reasons to generate fake short channel ids:
+/// 1) zero-conf channels that don't have a confirmed channel id yet
+/// 2) phantom node payments, to get an scid for the phantom node's phantom channel
+mod fake_scid {
```

Comment:
Maybe this module could be in `lightning/src/util/scid_utils.rs` ?

Reply:
I don't have a strong opinion here, nice to cut down on `channelmanager.rs` size though. @TheBlueMatt @jkczyz y'all fine with this?

Reply:
Sure, no opinion here. Make make #1295 easier/cleaner, even.